doc/source/data/monitoring-your-workload.rst

```diff
@@ -88,3 +88,4 @@ When an operator completes, the metrics for that operator are also logged.
    Operator InputDataBuffer[Input] -> TaskPoolMapOperator[ReadRange->MapBatches(<lambda>)] completed. Operator Metrics:
    {'num_inputs_received': 20, 'bytes_inputs_received': 46440, 'num_inputs_processed': 20, 'bytes_inputs_processed': 46440, 'num_outputs_generated': 20, 'bytes_outputs_generated': 800, 'rows_outputs_generated': 100, 'num_outputs_taken': 20, 'bytes_outputs_taken': 800, 'num_outputs_of_finished_tasks': 20, 'bytes_outputs_of_finished_tasks': 800, 'num_tasks_submitted': 20, 'num_tasks_running': 0, 'num_tasks_have_outputs': 20, 'num_tasks_finished': 20, 'obj_store_mem_alloc': 800, 'obj_store_mem_freed': 46440, 'obj_store_mem_cur': 0, 'obj_store_mem_peak': 23260, 'obj_store_mem_spilled': 0, 'block_generation_time': 1.191296085, 'cpu_usage': 0, 'gpu_usage': 0, 'ray_remote_args': {'num_cpus': 1, 'scheduling_strategy': 'SPREAD'}}
 
+This log file can be found locally at `/tmp/ray/{SESSION_NAME}/logs/ray-data.log`. It can also be found on the Ray Dashboard under the head node's logs.
```

Comment:
We want to make sure the first-time user can follow instrution to find the log. Can you add a link to https://docs.ray.io/en/latest/ray-observability/getting-started.html#logs-view ?